packages/boxel-ui/addon/src/components/radio-input/item.gts

```diff
@@ -124,10 +172,16 @@ const RadioInputItem: TemplateOnlyComponent<Signature> = <template>
   <label
     class={{cn
       'boxel-radio-option'
-      boxel-radio-option--checked=@checked
-      boxel-radio-option--disabled=@disabled
-      boxel-radio-option--hidden-border=@hideBorder
-      boxel-radio-option--has-radio=(not @hideRadio)
+      (if @checked 'boxel-radio-option--checked')
+      (if @disabled 'boxel-radio-option--disabled')
+      (if @hideBorder 'boxel-radio-option--hidden-border')
+      (if (not @hideRadio) 'boxel-radio-option--has-radio')
+      (if @variant (concat 'variant-' @variant) 'variant-default')
```

Comment:
you can add this line after line 174 to avoid having to make the changes to the other lines